### DISCUSSION TITLE
Projects (beta) Add PR/issue from different organization

### DISCUSSION THREAD

**Discussion by shakthimaan** [community/community#11256] [Projects and Issues]
Would it be possible to add a PR or issue from a different organization to an existing Projects (beta) board?

This will be useful for obtaining a bird's high-level view of the tasks in hand across organizations.

**Comment by RCheesley** [comment]
+1 for this, I'd actually like to be able to associate an issue/pr with an item, without having to create an item *from* the issue or PR. I guess that makes it complex as you can then maybe have more than one issue/PR potentially so it makes status automation a bit complicated.
